db/db.go

```diff
@@ -0,0 +1,457 @@
+/*
+Copyright 2025 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package db manages the googet state sqlite database.
+package db
```

Comment:
Suggest renaming this package to googetdb to free up using db as a variable name in all the places where you're currently using goodb.